### PR TITLE
Refactor: Unify Docker Build Blue Prints

### PR DESCRIPTION
**Pull request by reneme**
This unifies the `./build.sh` scripts for all currently available RedHat-based docker blue prints. Furthermore it adds a bit of documentation.

### CODE REVIEW

**Comment by jblomer** [comment]
Not bad. I think I can see the build system beautification :)

File: ci/docker/rhel_common/build.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+echo "checking yum installation..."
+check_yum_environment
+
+echo "setting up bootstrap repository..."
+cat > $YUM_REPO_CFG << EOF
+[$YUM_REPO_NAME]
+name=$SYSTEM_NAME base system packages
+baseurl=$REPO_BASE_URL
+gpgkey=$GPG_KEY_PATHS
+gpgcheck=0
```
Comment (jblomer) [inline ci/docker/rhel_common/build.sh]: Shouldn't this be 1?

Comment (reneme) [inline ci/docker/rhel_common/build.sh]: actually it should >.<

Comment (reneme) [inline ci/docker/rhel_common/build.sh]: Now it is again: #1123

**Pull request merged**